raft/src/main/java/org/apache/kafka/snapshot/SnapshotWriter.java

```diff
@@ -113,12 +113,11 @@ private void finalizeSnapshotWithFooter() {
      * Create an instance of this class and initialize
      * the underlying snapshot with {@link SnapshotHeaderRecord}
      *
-     * @param snapshot a lambda to create the low level snapshot writer
```

Comment:
This is an existing issue. Could we add the missing javadoc for supplier?

Reply:
Fixed by rebasing onto the latest trunk.